src/components/nav/movieSearchResult.tsx

```diff
@@ -1,24 +1,29 @@
+import {useModal} from '@/context/modal-context'
 import * as React from 'react'
 import {FC} from 'react'
 import './style.css'
 
-const MovieSearchResult: FC<{
+export const MovieSearchResult: FC<{
+  id: number
   title: string
-  poster_med: string
-  year: number
-  openModal(imdb: string): void
-  imdb: string
-}> = ({title, poster_med, year, openModal, imdb}) => (
-  <div className="search-result-item" onClick={() => openModal(imdb)}>
-    <div className="search-result-item-img">
-      <img src={poster_med} alt="films-boster" />
-    </div>
-    <div className="search-result-item-info">
-      <h1>
-        {title} ({year})
-      </h1>
-    </div>
-  </div>
-)
+  poster_path: string
+  release_date: number
```

Comment:
It's best practice to have camel case constants.

Reply:
Since this is from API. I aliased them to `camelCase`